oak_containers_launcher/src/lib.rs

```diff
@@ -153,6 +195,32 @@ impl Launcher {
             .ok_or_else(|| anyhow::anyhow!("trusted app address not set"))
     }
 
+    /// Gets the endorsed attestation evidence that the untrusted application can send to remote
+    /// clients, which will verify it before connecting.
+    pub async fn get_endorsed_evidence(&mut self) -> anyhow::Result<AttestationBundle> {
+        // If we haven't received an attestation evidence, wait for it.
+        if let Some(receiver) = self.attestation_evidence_receiver.take() {
+            // Set a timeout of 5 minutes, since we don't want to wait forever if the VM didn't
+            // start properly.
+            let sleep = tokio::time::sleep(tokio::time::Duration::from_secs(300));
```

Comment:
Can be cdone as future cleanup, but since we follow the same pattern here and with the ready notification we should think of a way to reduce the code duplication. For a start the timeout should be a constant, but perhaps we can also create a wrapper to wait with timeout (or perhaps tokio already has one).

Reply:
Added a constant